openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -514,20 +612,26 @@
     "team-type": "Team Type",
     "term": "Term",
     "test": "Test",
+    "test-connection": "Test Connection",
     "test-entity": "Test {{entity}}",
     "test-plural": "Tests",
     "test-suite": "Test Suite",
     "test-suite-status": "Test Suite Status",
+    "testing-connection": "Testing Connection",
     "tests-summary": "Tests Summary",
     "thread-count": "Thread Count",
+    "three-dash-symbol": "---",
+    "three-dots-symbol": "•••",
```

Comment:
I am not confident to have symbols on translation any specific reason for adding them